predeploy.py

```diff
@@ -30,31 +27,30 @@
 </html>
 \'\'\'\n\n"""
 
-pyfiles = ["requirements.txt", "Procfile"]
-
 app_name = os.environ["DASH_APP_NAME"]
 
-app_path = os.path.join("apps", APPNAME_TO_DIRECTORY.get(app_name, app_name))
-
 app_file_name = ""
 
 # get the name of the file used to run the app;
 # this is not always 'app.py'
 
-with open(os.path.join(app_path, "Procfile"), "r") as f:
+with open(os.path.join("/app", "Procfile"), "r") as f:
     contents = f.read().split(" ")
     for item in contents:
         if "server" in item:
             app_file_name = item.split(":")[0]
 
-full_app_path = os.path.join(app_path, app_file_name + ".py")
+if app_file_name == "":
+    print("Could not find app file")
+    sys.exit(1)
+
+full_app_path = os.path.join("/app", app_file_name + ".py")
```

Comment:
This extracts `index` from Procfile and tail it to find `index.py`. Not sure whether app with `index.py` is deployable. - We should test with `dash-salesforce-crm`

Reply:
Well `app` definitely was not it, hence the change. Still trying to make that app deployable :)

Reply:
Okay, seems with my changes, this app works, and the tag is there as expected :)